gapic/api/artman_bigtable.yaml

```diff
@@ -24,46 +24,57 @@ go:
   git_repos:
     go:
       location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
-      gapic_subpath: bigtable/apiv2
+      paths:
+        - bigtable/apiv2
     staging:
-      gapic_subpath: generated/go/google-cloud-bigtable-v2
+      paths:
+        - generated/go/google-cloud-bigtable-v2
 java:
   gapic_code_dir: ${REPOROOT}/artman/output/java/google-cloud-bigtable
   git_repos:
     java:
       location: git@github.com:GoogleCloudPlatform/google-cloud-java.git
-      gapic_subpath: google-cloud-bigtable
+      paths:
+        - google-cloud-bigtable
```

Comment:
It is indentation sensitive (very much so), but the indentation I am using here is correct.